src/test/java/menagerie/gui/ItemInfoBoxTest.java

```diff
@@ -0,0 +1,42 @@
+package menagerie.gui;
+
+import menagerie.model.menagerie.GroupItem;
+import menagerie.model.menagerie.ItemTestUtils;
+import menagerie.model.menagerie.MediaItem;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static junit.framework.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class ItemInfoBoxTest extends UITest {
+
+  private final File img = ItemTestUtils.WHITE_IMAGE_FILE;
+  private final int imgHeight = 22;
+  private final int imgWidth = 21;
+
+  @Test
+  void testSetMediaItem() throws InterruptedException {
+    TestItemInfoBox infoBox = new TestItemInfoBox();
+    MediaItem m = new MediaItem(null, 1, 1, img);
+
+    infoBox.setItem(m);
+    // wait for rendering to be done
+    Thread.sleep(UITest.SLEEP_TIME);
```

Comment:
generally, unit tests should always be independent from the environment. however, this test is dependent on the time dimension.

... but just as a general remark - should be fine in the scope of our project :)